package.json

```
@@ -32,16 +32,17 @@
   },
   "scripts": {
     "build": "tsc",
-    "lint": "eslint . --ext .js,.ts",
+    "lint": "npm run lint:types; npm run lint:js; npm run lint:docs",
+    "lint:js": "eslint . --ext .js,.ts",
     "lint:docs": "npx -y alex .",
+    "lint:types": "tsc --noEmit",
     "prebuild": "rm -rf dist/",
     "prepack": "npm run build",
     "prepare": "husky install",
     "pretest": "npm run lint",
     "prettier": "prettier --list-different \"./**/**.{js,ts,md}\"",
-    "prettier:write": "prettier --list-different --write \"./**/**.{js,ts,md}\"",
-    "test": "tsc; jest --coverage",
-    "test-watch": "tsc; jest --watch",
+    "prettier:fix": "prettier --list-different --write \"./**/**.{js,ts,md}\"",
+    "test": "vitest run --coverage",
```

see [this comment](https://github.com/readmeio/api/pull/691#discussion_r1290700363)

```suggestion
    "test": "vitest --coverage",
```